tests/cross_functional/conftest.py

```diff
@@ -922,7 +922,7 @@ def factory(number_of_buckets, bucket_types, cloud_providers):
         """
         type = dict()
         type["data"] = bucket_types["data"]
-        reduce_expiration_interval(interval=2)
+        reduce_expiration_interval(interval=1)
         logger.info("Changed noobaa lifecycle interval to 2 minutes")
```

Comment:
should this be 1 minute as per code change above?

Reply:
correct